support/tests/test_tealium_collect/TealiumCollectTests.swift

```diff
@@ -172,9 +170,11 @@ class TealiumCollectTests: XCTestCase {
         
     }
     
+    // This test sometimes fails during an all test run, but fine when run individually.
+    // TODO: Determine cause.
     func testValidSend() {
         
-        let validURL = "https://collect.tealiumiq.com/vdata/i.gif?tealium_library_version=1.0.0&tealium_session_id=someSessionId&tealium_library_name=swift&tealium_vid=someVID&tealium_random=someRandomNumber&event_name=eventName&tealium_account=account&tealium_profile=profile&tealium_environment=environment&tealium_visitor_id=someVisitorId&tealium_firstparty_visitor_id=someVisitorId"
+        let validURL = "https://collect.tealiumiq.com/vdata/i.gif?tealium_library_version=1.1.3&tealium_session_id=someSessionId&tealium_library_name=swift&tealium_vid=someVID&tealium_random=someRandomNumber&event_name=eventName&tealium_account=account&tealium_profile=profile&tealium_environment=environment&tealium_visitor_id=someVisitorId&tealium_firstparty_visitor_id=someVisitorId"
         
```

Comment:
This test doesn't actually care about the non-AS critical keys such as library_version.  This test is only checking that URL calls go out and are received correctly by AS (we don't get a non-200 or x-error).  It's not a great unit test however, as using mock objects would be better.